packages/access-api/src/service/index.js

```diff
@@ -33,7 +33,11 @@ export function service(ctx) {
       info: Server.provide(Space.info, async ({ capability, invocation }) => {
         const results = await ctx.models.spaces.get(capability.with)
         if (!results) {
-          return new Failure('Space not found.')
+          return {
+            error: true,
+            status: 404,
+            message: 'Space not found.',
```

Comment:
I agree it would be good to have actionable feedback in the error message. I defer to @hugomrdias on whether this is the best advice to give (accept the suggestion) or whether to deal with that in separate PR.